beacon-chain/blockchain/chain_info.go

```diff
@@ -179,6 +180,14 @@ func (s *Service) HeadGenesisValidatorRoot() [32]byte {
 	return s.headGenesisValidatorRoot()
 }
 
+// HeadETH1Data returns the eth1data of the current head state.
+func (s *Service) HeadETH1Data() *ethpb.Eth1Data {
```

Comment:
Add a test for this.

Reply:
https://github.com/prysmaticlabs/prysm/pull/6188